src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneSwitchConfig.java

```diff
@@ -0,0 +1,28 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.Accordion;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneSwitchConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Sends a notification when approaching the end of a lane,")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(name = "Farming only", desc = "Show the notification only when farming.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean farmingOnly = false;
```

Comment:
Why do we need this setting? why would anyone use this feature while not farming? either remove it entirely, or set it default enabled, or toggle the feature (also while not farming, default disabled)